src/app/pages/contact/form.js

```diff
@@ -115,33 +138,48 @@ export default function ContactForm() {
             <input type="hidden" name="user_id" value={userUuid} />
             <input type="hidden" name="support_context" value={bodyParams} />
             <label>
-                What is your question about?
+                <div className="label-text">
+                    What is your question about?
+                </div>
                 <DropdownSelect
-                    name="subject" options={options}
-                    onValueUpdate={onChangeSubject}
+                    name="subject"
+                    options={options}
+                    onValueUpdate={setSubject}
                 />
             </label>
             {
-                assignableSelected && <label>What Assignable topic in particular?
-                <DropdownSelect
-                    name="feature" options={assignableOptions}
-                />
+                assignableSelected && <label>
+                    <div className="label-text">
+                        What Assignable topic in particular?
+                    </div>
+                    <DropdownSelect
+                        name="feature" options={assignableOptions}
+                    />
                 </label>
             }
             <LabeledInputWithInvalidMessage showMessage={showInvalidMessages}>
-                Your Name
+                <div className="label-text">
+                    Your Name
+                </div>
                 <input name="name" type="text" size="20" required />
             </LabeledInputWithInvalidMessage>
             <LabeledInputWithInvalidMessage showMessage={showInvalidMessages}>
-                Your Email Address
+                <div className="label-text">
+                   Your Email Address
+                </div>
                 <input name="email" type="email" required />
             </LabeledInputWithInvalidMessage>
             <LabeledInputWithInvalidMessage className="auto-height" showMessage={showInvalidMessages}>
-                Your Message
+                <div className="label-text">
+                    Your Message
+                </div>
                 <textarea cols="50" name="description" rows="6" required />
             </LabeledInputWithInvalidMessage>
+            <div className="label-text">
+                Please add a screenshot or any other file that helps explain your request.
+            </div>
             <FileButton name="attachment" />
```

Comment:
Can we add some horizontal padding to the `file-name` text?
![image](https://github.com/user-attachments/assets/c22b3963-24a1-49d6-bd1d-7d19887efd74)

Reply:
I standardized the styling on this with the contact form.

Reply:
So fixing it there would fix it for both?

Reply:
Oh nevermind I wasn't looking at my updated local os-web instance